server/service/src/user_account.rs

```diff
@@ -181,17 +193,15 @@ impl<'a> UserAccountService<'a> {
     ) -> Result<UserAccount, VerifyPasswordError> {
         let repo = UserAccountRowRepository::new(self.connection);
         let user = match repo
-            .find_one_by_user_name(username)
+            .find_one_active_by_username(username)
             .map_err(VerifyPasswordError::DatabaseError)?
         {
             Some(user) => user,
             None => return Err(VerifyPasswordError::UsernameDoesNotExist),
         };
         // verify password
-        let valid = verify(password, &user.hashed_password).map_err(|err| {
-            error!("verify_password: {}", err);
```

Comment:
was there a reason you removed the error loggin~?

Reply:
No reason! Adding back logging